src/common/util/expenseFileUrls.ts

```diff
@@ -0,0 +1,7 @@
+import getConfig from 'next/config'
+
+const { publicRuntimeConfig } = getConfig()
+
+export function expenseFileUrl(fileId: string) {
+  return `${publicRuntimeConfig.API_URL}/expenses/download-file/${fileId}`
```

Comment:
You are moving the download logic to the backend with direct API calls.
That's a good idea.

I didn't realize this is possible :)